test/specs/commonTests/isConformant.tsx

```diff
@@ -31,19 +31,25 @@ export default (Component, options: any = {}) => {
   const { throwError } = helpers('isConformant', Component)
 
   const componentType = typeof Component
+  const wrappedComponent = mount(<Component {...requiredProps} />)
+
+  const createComponent = props => {
```

Comment:
this should be better named - with `create` used in method's name there is a reasonable assumption that new instance will be initialised - this is not the case as already created instance will be reused.

Reply:
Will try to come up with better name for this one.